packages/crypto-wallet-core/src/constants/tokens.ts

```diff
@@ -32,7 +32,7 @@ export let ETH_TOKEN_OPTS = {
   '0x2260fac5e5542a773aa44fbcfedf7c193bc2c599': {
     name: 'Wrapped Bitcoin',
     symbol: 'WBTC',
-    decimal: 9,
+    decimal: 8,
```

Comment:
Good catch on the decimal number. Shouldn't this be `decimals` per your other changes?

Reply:
yes, you're right! ty!